### DISCUSSION TITLE
공공데이터 api를 호출하여 정보를 제공할 때 캐싱처리를 통한 데이터 로딩속도 최적화

### DISCUSSION THREAD

**Discussion by bysoyeon** [100-hours-a-week/discussions#36] [Deepdive]
**배경**
개발 초기엔 api 로 호출한 정보를 RDB에 저장하지 않고 캐싱디비에만 저장하여 고객에게 제공하는 방법으로 진행하였으나, 이후 모든 정보를 api로 호출하여 일괄 RDB에 저장하여 고객에게 제공하는 방법으로 변경하였다. 

고객에게 데이터를 보다 빠른 속도로 제공하기 위해 캐싱디비를 사용하여 로딩속도를 개선하고 그 과정에서 학습한 내용을 기록하고자 한다.

**Comment by startupcodekr** [comment]
왜 성능 비교해야하는지 프로젝트 시나리오 기준으로 제목을 작성해주세요
